faq/index.html

```diff
@@ -45,25 +45,17 @@ <h6 class="arrow-r">Why would I want to use gRPC?</h6>
+                            <p>Google has been using a lot of the underlying technologies and concepts in gRPC for a long time. The current implementation is being used in several of Google’s cloud products and Google externally facing APIs. It is also being used by <a href="http://corner.squareup.com/2015/02/grpc.html">Square</a>,  <a href="https://github.com/Netflix/ribbon">Netflix</a>, <a href="https://blog.gopheracademy.com/advent-2015/etcd-distributed-key-value-store-with-grpc-http2/">CoreOS</a>, <a href="https://blog.docker.com/2015/12/containerd-daemon-to-control-runc/">Docker</a>, <a href="https://github.com/cockroachdb/cockroach">Cockroachdb</a>, <a href="https://github.com/CiscoDevNet/grpc-getting-started">Cisco</a>, <a href="https://github.com/Juniper/open-nti">Juniper Networks</a> and many other organizations and individuals.</p>
                         </div>
                     </li>
 
-<<<<<<< HEAD
-    <h4>Who’s using this and why?</h4>
-    <p>Google has been using a lot of the underlying technologies and concepts in gRPC for a long time. The current implementation is being used in several of Google’s cloud products like BigTable, PubSub, Cloud Storage and Google externally facing APIs. It is also being used by <a href="http://corner.squareup.com/2015/02/grpc.html">Square</a>, <a href="https://github.com/Netflix/ribbon">Netflix</a>, <a href="https://blog.gopheracademy.com/advent-2015/etcd-distributed-key-value-store-with-grpc-http2/">CoreOS</a>, <a href="https://blog.docker.com/2015/12/containerd-daemon-to-control-runc/">Docker</a>, <a href="https://github.com/cockroachdb/cockroach">Cockroachdb</a>, <a href="https://github.com/CiscoDevNet/grpc-getting-started">Cisco</a>, <a href="https://github.com/Juniper/open-nti">Juniper Networks</a> and many other organizations and individuals.</p>
 
-    <h4>What programming languages are supported?</h4>
-    <p>C++, Java, Objective-C, Python, Ruby, Go, C#, Node.js.
-    All of the above mentioned languages are now in GA and follow semantic versioning for new versions.</p>
-    <p>Java implementation supports Android and Objective-C is targeted at iOS.</p>
-=======
                     <li class="submenu">
                         <h6 class="arrow-r">What programming languages are supported?</h6>
                         <div class="submenu-content">
                             <p>C++, Java, Objective-C, Python, Ruby, Go, C#, Node.js, PHP.
-                            All of the languages are in active development on GitHub and we are welcoming contributions.</p>
-                            <p>Java implementation supports Android and Objective-C is targeted at iOS.</p>
+                            All of the above mentioned languages are now in GA and follow semantic versioning for new versions.</p>
+                            <p>Java implementation supports Android and Objective-C is targeted at iOS and are also at 1.0 now.</p>
```

Comment:
Would recommend not tying version releases here.. its one more item to update. 
We can use {{ site.data.config.grpc_release_branch }} if you want though..
How about
There are stable releases for Android and iOS (Objective-C) implementations of gRPC as well.